docs/UserGuide.md

```diff
@@ -166,9 +166,48 @@ AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.
 If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
 </div>
 
-### Archiving data files `[coming in v2.0]`
+## Event Management [Coming Soon!]
+Amigos also allows you to keep track of your social events!
+![Events Page](images/EventPage.png)
 
-_Details coming soon ..._
+### Creating an event: `addevent`
+Adds a new event, which can be optionally linked to existing friends.
+
+**Format**: `addevent n/{EVENT_NAME} dt/{DATE_TIME} dd/{DESCRIPTION} f/{FRIEND_NAME1, FRIEND_NAME2…}`
```

Comment:
Wondering about this. Brackets - `[]` vs `{}`. AddressBook used `[]`.

Reply:
I was following the formatting we used on the GDocs file - I think it should be fine so long as we standardize?